ch02_Context.md

```diff
@@ -115,7 +115,7 @@ die disciplines kunnen bruikbaar zijn. Met name het BiSL-raamwerk voor
 functioneel beheer is in enige mate bruikbaar, en deze is dan ook
 meegenomen in de totstandkoming van BOMOS (Voor meer informatie over
 BiSL: Best Practice - BiSL – Een framework voor Functioneel Beheer en
-Informatiemanagement , Remko van der Pols, Ralph Donatz, Frank van
+Informatiemanagement, Remko van der Pols, Ralph Donatz, Frank van
```

Comment:
geen spatie voor een comma

Reply:
Snap hem even niet, moet er wel een spatie voor de komma in dit geval?